### DISCUSSION TITLE
Backup is failing with velero and minio as backup storage

### DISCUSSION THREAD

**Discussion by SoundaryaA3098** [vmware-tanzu/velero#3321] [Community support Q&A] [answered]
Hi, I am trying to take backup of an AKS cluster with the pvc data using velero (v1.5.3). I have configured minio as backup storage location. After installed minio and velero, I created the backup using "velero backup create" command.

My backup is getting partiallyfailed with errors, velero backup logs gave me the following result,

An error occurred: Get "http://minio.minio.svc.cluster.local:9000/velero/backups/backup-v1/backup-v1-logs.gz?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=minio%2F20210122%2Fminio%2Fs3%2Faws4_request&X-Amz-Date=2021018987446Z&X-Amz-Expires=600&X-Amz-SignedHeaders=host&X-Amz-Signature=ae27cc376ad4691f6261477836544214d227e620d3f2944e194289c7b8b35": dial tcp: lookup minio.minio.svc.cluster.local on 127.0.0.53:53: server misbehaving.

I tested the connectivity of the minio from the other pod, it is reachable. Can anyone please guide me to solve the issue. Thanks in Advance!

**Comment by dsu-igeek** [comment, marked as answer]
In order to get the logs, velero backup logs will download the log file directly from the S3 server.  It looks like you have configured Minio within your cluster and are using the cluster local name.  That name will not resolve externally.  In order to make this work,
please define the publicURL parameter with either the IP address of the Minio installation or a publicly resolvable DNS name for the Minio installation.

An example is here:
https://velero.io/blog/velero-v1-1-on-vsphere
